### DISCUSSION TITLE
Add `section` property to action

### DISCUSSION THREAD

**Discussion by karniv00l** [TNorbury/command_palette#15] [Ideas]
Hi 👋 

I think it would be nice to be able to categorize actions with something like a `section`. Like in [kbar](https://github.com/timc1/kbar).

Example:

<img width="615" alt="Screenshot 2022-07-06 at 22 07 25" src="https://user-images.githubusercontent.com/3144291/177634059-4425039e-55a3-46d4-8269-8fae5277f2e9.png">

**Comment by TNorbury** [comment]
Yeah, good idea. I've made an issue: #21
